app/src/main/java/org/mozilla/fenix/ext/Activity.kt

```diff
@@ -22,3 +23,20 @@ fun Activity.enterToImmersiveMode() {
             or View.SYSTEM_UI_FLAG_FULLSCREEN
             or View.SYSTEM_UI_FLAG_IMMERSIVE_STICKY)
 }
+
+fun Activity.breadcrumb(
+    message: String,
+    data: Map<String, String> = emptyMap()
+) {
+    val activityName = this::class.simpleName ?: return
```

Comment:
Does `this:class.simpleName` work in a production build without kotlin reflection? We had to do this to get to the actual class name in migration: https://github.com/mozilla-mobile/android-components/pull/7897

`this::class.java` or `this.javaclass` work fine not needin Kotlin reflection.